lib/src/main/kotlin/at/bitfire/ical4android/DmfsTask.kt

```diff
@@ -65,26 +64,33 @@ import java.util.logging.Logger
  * The SEQUENCE field is stored in [Tasks.SYNC_VERSION], so don't use [Tasks.SYNC_VERSION]
  * for anything else.
  */
-abstract class DmfsTask(
-    val taskList: DmfsTaskList<DmfsTask>
+open class DmfsTask(
+    val taskList: DmfsTaskList<*>
 ) {
 
-    companion object {
-        const val UNKNOWN_PROPERTY_DATA = Properties.DATA0
-    }
-
     protected val logger = Logger.getLogger(javaClass.name)
     protected val tzRegistry by lazy { TimeZoneRegistryFactory.getInstance().createRegistry() }
 
     var id: Long? = null
+    var syncId: String? = null
+    var eTag: String? = null
+    var flags: Int = 0
 
 
-    constructor(taskList: DmfsTaskList<DmfsTask>, values: ContentValues): this(taskList) {
+    constructor(taskList: DmfsTaskList<*>, values: ContentValues): this(taskList) {
         id = values.getAsLong(Tasks._ID)
+        syncId = values.getAsString(Tasks._SYNC_ID)
+        eTag = values.getAsString(COLUMN_ETAG)
+        flags = values.getAsInteger(COLUMN_FLAGS) ?: 0
     }
 
-    constructor(taskList: DmfsTaskList<DmfsTask>, task: Task): this(taskList) {
+    constructor(taskList: DmfsTaskList<*>, task: Task, syncId: String?, eTag: String?, scheduleTag: String?, flags: Int): this(taskList) {
+        if (scheduleTag != null)
+            logger.fine("Schedule-Tag for tasks not supported yet, won't save")
```

Comment:
As the API changes anway and doesn't need to follow some DAVx5 scheme anymore, can we completely remove the `scheduleTag` from the arguments if it's not supported?